test/compilationTests/stringutils/strings.sol

```diff
@@ -688,7 +689,8 @@ library strings {
             return "";
 
         uint len = self._len * (parts.length - 1);
-        for(uint i = 0; i < parts.length; i++)
+        uint i;
```

Comment:
While this is not crucial, probably better style is to keep the variable declaration in each for loop.